conans/client/conf/__init__.py

```diff
@@ -103,7 +103,7 @@
         clang:
             version: ["3.3", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "4.0",
                       "5.0", "6.0", "7.0", "7.1",
-                      "8", "9", "10", "11", "12", "13", "14"]
+                      "8", "9", "10", "11", "12", "13", "14", "15"]
```

Comment:
Is there some reference, link, or have you checked that the the cppstd keeps being ``"gnu14"`` for clang 15? I would expect it, but just in case.

Reply:
- https://clang.llvm.org/docs/ReleaseNotes.html
- https://github.com/llvm/llvm-project/issues/53869